SU2_CFD/src/hybrid_RANS_LES_model.cpp

```diff
@@ -561,6 +557,11 @@ void CHybrid_Mediator::ComputeInvLengthTensor(CVariable* flow_vars,
   // 2) tauSGRS contribution.  NB: Neglecting divergence contribution
   // here.  TODO: Add divergence contribution.
 
+  /*--- Testing on the WMH indicates that scaling the whole stress by
+   * alpha*(2-alpha) improves the model performance.  That change would
+   * make the turbulent kinetic energy inconsistent, so it is avoided here.
+   * But that indicates there's some other issue. ---*/
+
   su2double alpha_fac = alpha*(2.0 - alpha);
```

Comment:
An update on this.  Sigfried indicates it isn't actually true.  I guess that conclusion (about scaling the whole stress by a*(2-a)) was based on looking at just one snapshot.  Anyway, now he says, "After many more runs, I was wrong about the scaling of the trace used in rM.  The best results come from alpha scaling on the isotropic part".